fxapom/pages/marionette/base.py

```diff
@@ -0,0 +1,16 @@
+
+from fxapom.fxapom import TIMEOUT
+from page import Page
+
+
+class Base(Page):
+    def __init__(self, driver, timeout=TIMEOUT):
+        super(Page, self).__init__(driver, timeout)
+        self.driver = driver
+        self._main_window_handle = self.driver.current_window_handle
+
+    def switch_to_main_window(self):
+        self.driver.switch_to_window(self._main_window_handle)
+
+    def close_window(self):
+        self.driver.close()
```

Comment:
I added it because you mentioned that I should likely leave the original page.py as is which has functionality in base.py that I can't change. Because of that, I just rewrote a marionette version of base and page.py